server/odc-service/src/main/java/com/oceanbase/odc/service/connection/model/ConnectionConfig.java

```diff
@@ -72,6 +72,9 @@ public class ConnectionConfig
     private static final String JDBC_URL_PARAMETERS_KEY = "JDBC_URL_PARAMETERS";
     private static final String CLOUD_PROVIDER = "cloudProvider";
     private static final String REGION = "region";
+    private static final String OB_INSTANCE_NICK_NAME = "instanceNickName";
```

Comment:
we may use these static name in odc-cloud, I notice that you use primitive value in odc-cloud